dev/cognitiveservices/data-plane/Language/analyzeconversations-authoring.json

```diff
@@ -1388,6 +1941,17 @@
           "items": {
             "$ref": "#/definitions/ConversationalAnalysisAuthoringExportedPrebuiltEntity"
           }
+        },
+        "regex": {
+          "description": "The regex component of the entity.",
+          "$ref": "#/definitions/ConversationalAnalysisAuthoringExportedEntityRegex"
+        },
+        "requiredComponents": {
+          "description": "The required components. Allowed values are 'learned', 'list' and 'prebuilts'.",
```

Comment:
I've seen this in a couple places in this PR: if values are limited, this should be an enum. Also set `modelAsString: true` to make sure any future values don't break callers.

Reply:
@heaths the reason string is used instead of enum is that in the future this can be extended to allow specific entity component names, which will be custom values provided by the user. Also, this is just a copy paste of what we already have in our GA 2022-05-01 APIs so we can't break it.